crates/ui/src/components/context_menu.rs

```diff
@@ -356,12 +443,17 @@ impl Render for ContextMenu {
     fn render(&mut self, cx: &mut ViewContext<Self>) -> impl IntoElement {
         let ui_font_size = ThemeSettings::get_global(cx).ui_font_size;
 
-        div().occlude().elevation_2(cx).flex().flex_row().child(
-            WithRemSize::new(ui_font_size).flex().child(
+        WithRemSize::new(ui_font_size)
+            .occlude()
+            .elevation_2(cx)
+            .flex()
+            .flex_row()
+            .child(
                 v_flex()
                     .id("context-menu")
                     .min_w(px(200.))
                     .max_h(vh(0.75, cx))
+                    .flex_1()
```

Comment:
This is needed to have the context menu fill the parent container. We look at many context menus throughout the app after this change and it didn't seem to affect them.